modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -70,20 +41,20 @@ You can also change the encryption algorithm and crypto mode by setting addition
 | The type of algorithm you use to encrypt the content of the property. See <<supported_algorithms>> for a complete list.
 
 | Mode
-| The procedure that allows the Mule runtime to repeatedly use a block cipher with a single key. See <<supported_modes>> for a full list.
+| The procedure that allows the Mule runtime engine to repeatedly use a block cipher with a single key. See <<supported_modes>> for a full list.
 |===
 
-[[supported_files]]
-== Supported Files
```

Comment:
This section might be kept with just the supported file types, something like this maybe:

== Supported Configuration File Types

* Spring-formatted properties: `.properties`
* YAML: `.yaml`

If this section is added, I think it should be next to `== Supported Algorithms and Modes`

Reply:
Added